hydra-cluster/src/Hydra/Cluster/Scenarios.hs

```diff
@@ -1515,6 +1515,89 @@ canRecoverDeposit tracer workDir backend hydraScriptsTxId =
  where
   hydraTracer = contramap FromHydraNode tracer
 
+-- | Open a single participant head, deposit, Close and then recover it.
+canRecoverDepositWhenClosed :: ChainBackend backend => Tracer IO EndToEndLog -> FilePath -> backend -> [TxId] -> IO ()
+canRecoverDepositWhenClosed tracer workDir backend hydraScriptsTxId =
```

Comment:
:+1: This is a good acceptance test for the issue at hand (https://github.com/cardano-scaling/hydra/issues/1812)